presto-hive/src/test/java/com/facebook/presto/hive/TestCteExecution.java

```diff
@@ -1242,14 +1261,28 @@ private void compareResults(MaterializedResult actual, MaterializedResult expect
     {
         // Verify result count
         assertEquals(actual.getRowCount(),
-                expected.getRowCount(), String.format("Expected %d rows got %d rows", expected.getRowCount(), actual.getRowCount()));
+                expected.getRowCount(), format("Expected %d rows got %d rows", expected.getRowCount(), actual.getRowCount()));
         if (checkOrdering) {
             assertEquals(actual.getMaterializedRows(), expected.getMaterializedRows(), "Correctness check failed! Rows are not equal");
             return;
         }
         QueryAssertions.assertEqualsIgnoreOrder(actual, expected, "Correctness check failed! Rows are not equal");
     }
 
+    private void checkCTEInfoMatch(String explain, List<CTEInfoValue> expectedCTEInfoValues)
+    {
+        String regex = "CTEInfo.*";
+        Pattern pattern = Pattern.compile(regex);
```

Comment:
move this to a constant to avoid recomputing

Reply:
Good idea - since latest changes.

Reply:
https://github.com/prestodb/presto/blob/master/CONTRIBUTING.md
our developer guide does seem to prefer immutable list